src/core/properties/cameraproperty.cpp

```diff
@@ -56,16 +56,22 @@ CameraProperty::CameraProperty(const std::string& identifier, const std::string&
     , cameraType_("cameraType", "Camera Type",
                   InviwoApplication::getPtr()->getCameraFactory()->getKeys(), 0)
     , cameraActions_("actions", "Actions", buttons())
-    , lookFrom_("lookFrom", "Look from", eye, -vec3(100.0f), vec3(100.0f), vec3(0.1f),
+    , lookFrom_("lookFrom", "Look from", eye, {-vec3(100.0f), ConstraintBehaviour::Ignore},
+                {vec3(100.0f), ConstraintBehaviour::Ignore}, vec3(0.1f),
                 InvalidationLevel::InvalidOutput, PropertySemantics{"SphericalSpinBox"})
-    , lookTo_("lookTo", "Look to", center, -vec3(100.0f), vec3(100.0f), vec3(0.1f),
+    , lookTo_("lookTo", "Look to", center, {-vec3(100.0f), ConstraintBehaviour::Ignore},
+              {vec3(100.0f), ConstraintBehaviour::Ignore}, vec3(0.1f),
               InvalidationLevel::InvalidOutput, PropertySemantics::SpinBox)
-    , lookUp_("lookUp", "Look up", lookUp, -vec3(1.f), vec3(1.f), vec3(0.1f),
+    , lookUp_("lookUp", "Look up", lookUp, {-vec3(1.f), ConstraintBehaviour::Immutable},
+              {vec3(1.f), ConstraintBehaviour::Immutable}, vec3(0.1f),
               InvalidationLevel::InvalidOutput, PropertySemantics::SpinBox)
-    , aspectRatio_("aspectRatio", "Aspect Ratio", 1.0f, 0.0f, std::numeric_limits<float>::max(),
-                   0.01f, InvalidationLevel::InvalidOutput, PropertySemantics::Text)
-    , nearPlane_("near", "Near Plane", 0.1f, 0.001f, 10.f, 0.001f)
-    , farPlane_("far", "Far Plane", 100.0f, 1.0f, 1000.0f, 1.0f)
+    , aspectRatio_("aspectRatio", "Aspect Ratio", 1.0f, {0.0f, ConstraintBehaviour::Immutable},
+                   {std::numeric_limits<float>::max(), ConstraintBehaviour::Ignore}, 0.01f,
```

Comment:
We can reduce the max bound now that we can ignore it. A reasonable value might be 16/9?

Reply:
this has been removed.